test/integration/winrm_test.rb

```diff
@@ -0,0 +1,196 @@
+# encoding: utf-8
```

Comment:
Its safe to remove these lines, we're using a Ruby > 2.0

Reply:
Yes please.